jax/config.py

```diff
@@ -160,3 +164,9 @@ def __getattr__(self, name):
     bool_env('JAX_OMNISTAGING', True),
     help='Enable staging based on dynamic context rather than data dependence.'
 )
+
+flags.DEFINE_integer(
+    'jax_traceback_num_frames',
+    int_env('JAX_TRACEBACK_NUM_FRAMES', 3),
+    help='Set the number of stack frames in JAX tracer error messages.'
+)
```

Comment:
"Tracebacks" exist in contexts outside of JAX tracer error messages. Could we make this flag name more precise (say, `jax_tracer_error_traceback_frames`), or do we plan to respect it more broadly somehow?

Reply:
Good idea!